src/main/java/seedu/trackbeau/ui/StatisticsPanel.java

```diff
@@ -0,0 +1,17 @@
+package seedu.trackbeau.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.trackbeau.model.customer.Customer;
+
+public class StatisticsPanel extends UiPart<Region> {
```

Comment:
maybe can add documentation?

Reply:
oo like?

Reply:
I think she meant java doc for the class.